sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/DigitalTwinsLifecycleSample.java

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.digitaltwins.core;
+
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.digitaltwins.core.implementation.models.ErrorResponseException;
+import com.azure.identity.ClientSecretCredentialBuilder;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.concurrent.Semaphore;
+import java.util.concurrent.TimeUnit;
+
+public class DigitalTwinsLifecycleSample {
+    private static final ClientLogger logger = new ClientLogger(DigitalTwinsLifecycleSample.class);
+
+    private static final String tenantId = System.getenv("TENANT_ID");
+    private static final String clientId = System.getenv("CLIENT_ID");
+    private static final String clientSecret = System.getenv("CLIENT_SECRET");
+    private static final String endpoint = System.getenv("DIGITAL_TWINS_ENDPOINT");
+
+    private static final int MaxTimeForTwinOperationsInSeconds = 10;
+
+    private static final URL DtdlDirectoryUrl = DigitalTwinsLifecycleSample.class.getClassLoader().getResource("DTDL");
+    private static final Path DtDlDirectoryPath;
+    private static final Path TwinsPath;
+    private static final Path ModelsPath;
+    private static final Path RelationshipsPath;
+
+    private static final DigitalTwinsAsyncClient client;
+
+    static {
+        try {
+            assert DtdlDirectoryUrl != null;
+            DtDlDirectoryPath = Paths.get(DtdlDirectoryUrl.toURI());
+        } catch (URISyntaxException e) {
+            logger.error("Unable to convert the DTDL directory URL to URI: ", e);
+            throw new RuntimeException(e);
+        }
+        TwinsPath = Paths.get(DtDlDirectoryPath.toString(), "DigitalTwins");
+        ModelsPath = Paths.get(DtDlDirectoryPath.toString(), "Models");
```

Comment:
ModelPath and RelationshipsPath is not used here.

Reply:
It will be, in the rest of the sample (this sample creates models and relationships) :)